docs/reference/api.rst

```diff
@@ -260,6 +329,31 @@ and stderr streams, is sent directly to the terminal.
     .. seealso::
         `SetBlocking` for configuring a global default blocking value
 
+    .. _autosave_arg:
+
+    `autosave`
```

Comment:
Ah okay, will try and fix. Using ".. _autosave" here creates warnings on a build because of the ambiguity between references to the autosave arg here and the softioc.autosave module in the docs.